### ISSUE TITLE
Add cryptocurrenciesAccepted property to LocalBusiness

### ISSUE THREAD

**Issue by ayumihamsaki** [schemaorg/schemaorg#1866]
Many businesses now accept cryptocurrencies as real payment exchanges. I am proposing to add the property: "`cryptocurrenciesAccepted`" to the "`LocalBusiness`".

You can use the same  ISO 4217 currency format as for `currenciesAccepted`. For example bitcoin would be "BTC". Because all cryptocurrencies use a Trading Ticker exactly the same as currencies.

I also propose updating "`paymentAccepted`" and adding "`Cryptocurrency`" to the valid list.

An example code is below:

```
<script type="application/ld+json">
{
        "@context": "http://schema.org",
        "@type": ["TouristAttraction", "AmusementPark"],
        "name": "Disneyland Paris",
        "description": "It's an amusement park in Marne-la-Vallée, near Paris, in France and is the most visited theme park in all of France and Europe.",
        "openingHours":["Mo-Fri 10:00-19:00", "Sa 10:00-22:00", "Su 10:00-21:00"],
        "isAccessibleForFree": false,
        "currenciesAccepted": "EUR",
        "cryptocurrenciesAccepted": [ "BTC", "ETH" ],
        "paymentAccepted":"Cash, Credit Card, Cryptocurrency",
        "url":"http://www.disneylandparis.it/"
}
</script>
```

I look forward to the schema team looking into this proposal in more detail.

Many Thanks.

**Comment by magico** [comment]
Is there a reason why `BTC` and `ETH` can not come up under  `currenciesAccepted` ? Technically they are currencies and I don't see a benefit from separating them.

**Comment by ayumihamsaki** [comment]
Central Banks and the World bank won't accept cryptocurrencies as normal currencies. So cryptocurrencies are classed as a different asset for legal reasons. That's why I suggested a new property called: "cryptocurrenciesAccepted"

**Comment by ayumihamsaki** [comment]
Central Banks and the World Bank won't accept cryptocurrencies as normal currencies. So cryptocurrencies are classed as a different asset for legal reasons. That's why I suggested a new property called: "cryptocurrenciesAccepted"

See here for more details about that according to the G20: https://www.bloomberg.com/news/articles/2018-03-20/when-is-a-bitcoin-not-a-bitcoin-when-it-s-an-asset-says-g-20

"Countries increasingly see cryptos as an asset, not a currency"

**Comment by RichardWallis** [comment]
@ayumihamsaki I understand that in the eyes of some banks etc cyptocurrencies are viewed differently. However, in the use case of a user identifying if a business accepts the currency they prefer, it is exactly the same as for 'normal' currencies.

If a business wants to highlight specific types of payment they accept by category (cash, card, etc.,) they could always use the [paymentAccepted](http://schema.org/paymentAccepted) property.

If a banking organization wish to highlight that they handle cyptocurrencies, they should describe it as one of the ```Service``` they provide.

**Comment by ayumihamsaki** [comment]
@magico @RichardWallis I see what you both are saying so you both are suggesting users to write the code like this:

```
<script type="application/ld+json">
{
        "@context": "http://schema.org",
        "@type": ["TouristAttraction", "AmusementPark"],
        "name": "Disneyland Paris",
        "description": "It's an amusement park in Marne-la-Vallée, near Paris, in France and is the most visited theme park in all of France and Europe.",
        "openingHours":["Mo-Fri 10:00-19:00", "Sa 10:00-22:00", "Su 10:00-21:00"],
        "isAccessibleForFree": false,
        "currenciesAccepted": [ "EUR", "BTC", "ETH" ],
        "paymentAccepted":"Cash, Credit Card, Cryptocurrency",
        "url":"http://www.disneylandparis.it/"
}
</script>
```

If that is fine then the documentation on the schema.org website can be updated to reflect accepting cryptocurrencies found here: [LocalBusiness](http://schema.org/LocalBusiness)

Something like:

![schema](https://user-images.githubusercontent.com/17784082/37916949-128e1c40-3115-11e8-8a12-a1c75c27a175.png)

**Comment by ayumihamsaki** [comment]
@RichardWallis Hi I was just looking at the work you are doing and I saw this in the commits:

`<span property="rdfs:comment">The currency in which the monetary amount is expressed (in 3-letter [ISO 4217](http://en.wikipedia.org/wiki/ISO_4217) format - For cryptocurrencies use the (ticker symbol)[https://en.wikipedia.org/wiki/List_of_cryptocurrencies]).</span>`

May I suggest not using the Wikipedia link as it is not very good and well maintained a much better link would be here: http://list.wiki/Cryptocurrencies

So my suggestion is this:

`<span property="rdfs:comment">The currency in which the monetary amount is expressed (in 3-letter [ISO 4217](http://en.wikipedia.org/wiki/ISO_4217) format - For cryptocurrencies use the (ticker symbol)[http://list.wiki/Cryptocurrencies]).</span>`

~If that is useful or not?

**Comment by sopekmir** [comment]
@ayumihamsaki , @RichardWallis: Perhaps we could consider cryptocurrencies as a "parallel" property to our existing http://schema.org/currency of the financial extension (http://schema.org/docs/financial.html) and it would become the property for types like DatedMoneySpecification,  ExchangeRateSpecification, MonetaryAmount (TBD)...
What do you think?

**Comment by RichardWallis** [comment]
@ayumihamsaki That looks like an interesting link.

Before proposing using that, I would want to find out, with regard to the list.wiki site, how stable it is, who hosts it, who is behind it, how active its community is, etc.  I could not ascertain this from browsing the site, or Wikipedia, or the web in general.

I would also personally comment that considering the writeup on the site makes much emphasis about complementing, and making the information it contains available to, search engines, I am a little surprised that it does not yet publish Schema.org in its pages.

**Comment by RichardWallis** [comment]
@sopekmir I think a separate cryptocurrency property is unnecessary. The currency property is capable of holding a mixture of types.  All that is needed is relevant guidance in the term definition.

My current proposed version for this is:

> The currency accepted.
> 
> Use standard formats: [ISO 4217 currency format](http://en.wikipedia.org/wiki/ISO_4217) e.g. "USD"; [Ticker symbol](https://en.wikipedia.org/wiki/List_of_cryptocurrencies) for cryptocurrencies e.g. "BTC"; well known names for [Local Exchange Tradings Systems](https://en.wikipedia.org/wiki/Local_exchange_trading_system) (LETS) and other currency types e.g. "Ithaca HOUR".
>

**Comment by sopekmir** [comment]
@RichardWallis - I agree. Seems I just did not remember that the range of the currency property is text and can take values from other sources than ISO 4217. And as far as I see - you added the references to the cryptocurrencies for this and other relevant properties. So this seems to be resolved. Is it?

**Comment by RichardWallis** [comment]
Released in V3.4
